api/web.config

```diff
@@ -13,6 +13,10 @@
           <match url="^version/(1.0|2.0)$" />
           <action type="Rewrite" url="versioninfo.php?version={R:1}" appendQueryString="true" />
         </rule>
+        <rule name="version/all" stopProcessing="true">
+          <match url="^version/all$" />
+          <action type="Rewrite" url="versioninfoall.php" appendQueryString="true" />
+        </rule>
```

Comment:
Is web.config still being used? If not maybe we remove on a separate PR.

Reply:
Still used for test dev. We don't have a dockerized version of this site.